7 geom sf label

jengelaere opened the pull request:
- grosse mise à jour renv pour éviter que nos projets ne se fossilisent
 - ajout de l'exemple geom_sf_label #7 
 - correction bug facette tmaps #8 
 - vérifications et apport de certaines modifs listées #3 
 - ajout scale_xx_identity #2 
Corrections savoir fR correspondantes réalisées sur la branche master désolée 😞

MurielleDelmotte committed:
Merge pull request #17 from MTES-MCT/master

Mise à niveau dev par rapport à master

jengelaere committed:
Merge pull request #20 from MTES-MCT/master

Mise à niveau branche de dev

jengelaere committed:
Rétablissement des facettes tmap - Closes #8

jengelaere committed:
Ajout exemple geom_sf_label()

jengelaere committed:
closes #7 - closes #8 - supression renv

jengelaere committed:
maj packages et réactivation renv

jengelaere committed:
ajout fonctions scales_xx_identity (closes #2)
maj synthaxe facettes, remplacement geom_bar par geom_col pour éviter de recourir à stat

closes #19 closes #2

jengelaere committed:
Contributes to #3
- Ajout lien vers aide theme() plus explicite
- Correction exo 3 quote argument type de scale_color_brewer
- mise à jour des consignes du chapitre exercice avec celles disséminées dans le support
- instruction installation addin colour picker

jengelaere committed:
Contributes to #3
- Ajout lien vers aide theme() plus explicite
- Correction exo 3 quote argument type de scale_color_brewer
- mise à jour des consignes du chapitre exercice avec celles disséminées dans le support
- instruction installation addin colour picker

jengelaere committed:
Merge branch '7-geom_sf_label' of github.com:MTES-MCT/parcours_r_module_datavisualisation into 7-geom_sf_label

jengelaere committed:
Rajout highcharter :cry:

MurielleDelmotte commented:
@jengelaere 
Dans le rendu de l'exo 4 partie `10-les-facettes.Rmd` et la correction de l'exo 4 partie `16-exercices.Rmd`, tu as remplacé le `geom_bar `par un `geom_col`, le résultat est identique.
Cependant on ne présente pas `geom_col `dans les formes géométriques ( partie `04-les-formes_geometriques.Rmd`) et dans la partie théorie des facettes on utilise encore le `geom_bar`....
Je pencherais pour la cohérence entre les parties 🙄

jengelaere commented:
Tu as raison Murielle, désolée, je revois ma copie

jengelaere committed:
Présentation geom_col() : ajout d'une ligne au tableau des geom, ajout un § sur les différences entre geom_bar et geom_col, ajout d'un exemple.

Ajout des parametres aesthetic possibles aux geom préexistants (à partir de la feuille de triche ggplot2)

Homogénéisation recours à geom_col au lieu de geom_bar(aes(weight = variable))

Nouvelle synthaxe de facet_wrap pour les derniers exemples qui n'avaient pas été modifiés

:warning: Passage de l'addin styler pour ajouter les espaces manquants (scope = "indention") Cocher la case Ignore whitespace pour voir les modifications réellement apportées au support.

Le book se compile normalement chez moi? Renv ok :
```
> renv::status()
* The project is already synchronized with the lockfile.
```

jengelaere commented:
- Présentation geom_col() : 
  -   ajout d'une ligne au tableau des geom, 
  -   ajout un § sur les différences entre geom_bar et geom_col, 
  -   ajout d'un exemple.
- Ajout des parametres aesthetic possibles aux geom préexistants (à partir de la feuille de triche ggplot2)
- Homogénéisation recours à geom_col au lieu de geom_bar(aes(weight = variable))
- Nouvelle synthaxe de facet_wrap pour les derniers exemples qui n'avaient pas été modifiés

⚠️ Passage de l'addin styler pour ajouter les espaces manquants (scope = "indention") 
Je vous recommande de cocher la case Ignore whitespace pour voir les modifications réellement apportées au support.

Le book se compile normalement chez moi. Renv ok :
```
renv::status()
# The project is already synchronized with the lockfile.
```

jengelaere committed:
Merge branch 'master' into 7-geom_sf_label
